go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/pganalyze/pg_query_go/v2
+module github.com/pganalyze/pg_query_go/v4
```

Comment:
I've been wondering if bumping up the major version of the Go module is the right thing to do.

It does lead to a lot of churn on the library being imported (i.e. one has to update all the files), but there are indeed some backwards-incompatible changes in the parsetree due to the Value node change.

If I recall correctly, the main reason why I bumped up from "v1" to v2 a while ago was because we switched the parse tree result to be auto-generated (based on Protobufs) and not JSON, and that change felt big enough to require a clean break.

That said, its a new major version for Postgres, and so that'd make a new major version of the library seem correct as well (and in a sense that's how we've been treating libpg_query itself).

@msakrejda @seanlinsley @keiko713 Any thoughts / opinions on this?

Reply:
I think we should bump. Having to explicitly change some package references to upgrade is annoying, but breaking changes on upgrading to a new minor version are a lot more annoying. Plus, there's `gofmt -r` or `sed` to help with the former.